CHANGELOG.md

```diff
@@ -1,9 +1,15 @@
 # Optimizely Ruby SDK Changelog
 
+## 3.10.1
+February 2, 2022
+
+### Enhancements:
+- Generate OptimizelyConfig object on API Call instead of SDK initialization to make the initialization faster (([#296](https://github.com/optimizely/ruby-sdk/pull/296)).
```

Comment:
An error in the PR link reference format.

Reply:
Good catch! just fixed it.